src/App.tsx

```diff
@@ -17,18 +76,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                filter={filterByQuery}
+                onClose={clearQuery}
```

Comment:
```suggestion
                onReset={clearQuery}
```
I would name it like this better